Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -185,6 +194,57 @@ public void PostProc_ValidArgsPassedThrough()
         VerifyTargetsUninstaller();
     }
 
+    [TestMethod]
+    public void PostProc_ScannerEngine_Success()
+    {
+        config.HasBeginStepCommandLineCredentials = true;
+        config.EngineJarPath = "engine.jar";
+        config.UseSonarScannerCli = false;
+
+        Execute(["/d:sonar.token=token"]).Should().BeTrue("Expecting post-processor to have succeeded");
+
+        scanner.DidNotReceiveWithAnyArgs().Execute(null, null, null);
+        engine.Received(1).Execute(config, """
+                {
+                  "scannerProperties": [
+                    {
+                      "key": "sonar.modules",
+                      "value": ""
+                    }
+                  ]
+                }
+                """
+            .ToEnvironmentLineEndings());
```

Comment:
The styling rule has a FP here. The raw string literal should be +4 from the `engine` (-4 from current position), and the `.ToEnvironmentLineEndings()` should be +4 from it's expression (+4 from current position).

It will show FP